proxyclient/m1n1/proxyutils.py

```diff
@@ -86,17 +86,22 @@ def __init__(self, p, heap_size=1024 * 1024 * 1024):
         }
 
     def read(self, addr, width):
+        '''read(addr, width) - do a width read from addr and return it
+        width can be 8,16,21,64 or 132'''
         val = self._read[width](addr)
         if self.proxy.get_exc_count():
             raise ProxyError("Exception occurred")
         return val
 
     def write(self, addr, data, width):
+        '''write(addr, data, width) - do a width write of data to addr
+        width can be 8,16,21,64 or 132'''
         self._write[width](addr, data)
         if self.proxy.get_exc_count():
             raise ProxyError("Exception occurred")
 
     def mrs(self, reg, *, silent=False, call=None):
+        '''mrs(reg) - read system register reg'''
```

Comment:
Perhaps mention that this takes a bunch of different sysreg styles as parsed by `sysreg_parse`: a tuple representing the encoding, a string identifier in "sN_N_cN_cN_N" form, or a register name if in the database.

Reply:
Addressed in latest commit